service/src/main/java/bio/terra/externalcreds/controllers/AdminApiController.java

```diff
@@ -67,6 +69,16 @@ public ResponseEntity<AdminLinkInfo> getLinkedAccountForExternalId(
     return ResponseEntity.of(linkedAccount.map(OpenApiConverters.Output::convertAdmin));
   }
 
+  @Override
+  public ResponseEntity<List<Object>> getVisas(
+      Provider provider, String userId, String visaType, String issuer) {
+    requireAdmin();
+    return ResponseEntity.ok(
+        passportService.getVisaClaims(provider, userId, issuer, visaType).stream()
```

Comment:
nit/optional: the arguments for `AdminApiController.getVisas()` and `PassportService.getVisaClaims()` swap the order of `visaType` and `issuer`. Since these are both Strings, it feels easy to make a mistake here; would be nice to align the argument order. That said, I realize `PassportService.getVisaClaims()` is only called from this one place and this isn't a particularly big deal; your call on changing it or not.